files/get_hiera/1/hieradata/common.yaml

```diff
@@ -1,2 +1,2 @@
 ---
-message: "Welcome to $::hostname. Don't break anything!"
+message: "Welcome to try.puppet.com. Don't break anything!"
```

Comment:
Yeah, I know. This is the first lesson so I wanted to make it simpler. Also the interpolation doesn't work with the command line hiera unless you say `hiera hostname=something`